app/Helpers/SearchHelper.php

```diff
@@ -44,7 +44,7 @@ public static function searchContacts(string $needle, string $orderByColumn, str
             });
 
             return $b->addressBook($accountId, $addressBookName)
```

Comment:
`$b` is not a great variable name (even though it wasn't written in this PR).

Reply:
I've fixed here